src/Analysis/Ast/Impl/Analyzer/PythonAnalyzerEntry.cs

```diff
@@ -219,9 +218,14 @@ public Task<IDocumentAnalysis> GetAnalysisAsync(CancellationToken cancellationTo
                 }
 
                 UpdateAnalysisTcs(analysisVersion);
-                dependencies = _parserDependencies != null
-                    ? ImmutableArray<AnalysisModuleKey>.Create(_parserDependencies.Union(_analysisDependencies).ToArray())
-                    : ImmutableArray<AnalysisModuleKey>.Create(_analysisDependencies);
+                if (_analysisDependencies != null) {
+                    dependencies = _parserDependencies != null
+                        ? ImmutableArray<AnalysisModuleKey>.Create(_parserDependencies.Union(_analysisDependencies).ToArray())
+                        : ImmutableArray<AnalysisModuleKey>.Create(_analysisDependencies);
+                } else if(_parserDependencies != null) {
```

Comment:
Nit: formatting

Reply:
whats wrong with it?

Reply:
`if(` vs `if (`. Minor (but VS normally fixes it).